src/app/ui/query-tabs/tabs.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TabsComponent } from './tabs.component';
+
+describe('TabsComponent', () => {
```

Comment:
Let's remove redundant spec files. You can set default options for component generation in nx project json file